README.md

```diff
@@ -1,36 +1,114 @@
-# Conduit Connector for <resource>
-[Conduit](https://conduit.io) for <resource>.
+# Conduit Connector for Activemq Artemis
+
+The ActiveMQ Classic connector is one of [Conduit](https://conduit.io) plugins. The connector provides both a source and a destination connector for [ActiveMQ Artemis](https://activemq.apache.org/components/artemis/).
+
+It uses the [stomp protocol](https://stomp.github.io/) to connect to ActiveMQ.
+
+## What data does the OpenCDC record consist of?
+
+| Field                   | Description                                                                          |
+|-------------------------|--------------------------------------------------------------------------------------|
+| `record.Position`       | json object with the destination and the messageId frame header.                     |
+| `record.Operation`      | currently fixed as "create".                                                         |
+| `record.Metadata`       | a string to string map, with keys prefixed as `activemq.header.{STOMP_HEADER_NAME}`. |
+| `record.Key`            | the messageId frame header.                                                          |
+| `record.Payload.Before` | <empty>                                                                              |
+| `record.Payload.After`  | the message body                                                                     |
+
 
 ## How to build?
 Run `make build` to build the connector.
 
 ## Testing
-Run `make test` to run all the unit tests. Run `make test-integration` to run the integration tests.
+Run `make test` to run all the tests. The command will handle starting and stopping docker containers for you.
+
+## Configuration
+
+Both source and destination connectors share the following parameters:
```

Comment:
As in the other connector, I'd mention these explicitly in both, the source and destination configuration.